### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by camilonova**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -2,6 +2,14 @@
 Changelog
 =========
 
+12.6.0
+======
+
+* Add Cropbox feature in Wand/Convert Engine
+* Add testing for Django 2.2
+* Remove "django.utils.six" to support Django 3.0+
```
Comment (benjaoming) [inline CHANGES.rst]: I think you should add something like "Drop Python 2 support"

Comment (benjaoming) [inline CHANGES.rst]:
```suggestion
* Remove "django.utils.six" to support Django 3.0+
* Remove Python 2 support
```

**Comment by camilonova** [comment]
@benjaoming can you approve it so i can make the release?

**Comment by camilonova** [comment]
Can anyone approve this?

**Comment by benjaoming** [comment]
@camilonova we need a review approval of someone with write access to the repo. Which unfortunately isn't me.

@wm3ndez or @timgraham can maybe help :)

**Comment by benjaoming** [comment]
@camilonova trying this instead: See if you can approve and merge #605

**Pull request closed without merging**